packages/gafl-webapp-service/src/handlers/__tests__/renewals-friendly-url-handler.spec.js

```diff
@@ -30,19 +30,21 @@ describe('The url handler', () => {
       [UTM.CAMPAIGN]: 'renewals',
       [QUERYSTRING_LICENCE_KEY]: null
     }
+    const request = generateRequestMock(params)
     const responseToolkit = generateResponseToolkitMock()
-    await urlHandler(generateRequestMock(params), responseToolkit)
-    expect(responseToolkit.redirect).toHaveBeenCalledWith(IDENTIFY.uri)
+    await urlHandler(request, responseToolkit)
+    expect(responseToolkit.redirectWithLanguageCode).toHaveBeenCalledWith(IDENTIFY.uri)
   })
 
   it.each([['B2F11U'], ['AH56F6'], ['GH330P']])('6 digit reference number exists and returns ATTRIBUTION', async licenceKey => {
     const params = {
       [QUERYSTRING_LICENCE_KEY]: licenceKey
     }
+    const request = generateRequestMock(params)
     const responseToolkit = generateResponseToolkitMock()
-    await urlHandler(generateRequestMock(params), responseToolkit)
+    await urlHandler(request, responseToolkit)
```

Comment:
I've noticed you doing this in a couple of places. A pattern I use is when there's no further use for a mock, I define it inline. I suppose by this logic we should do 
```
await urlHandler(generateRequestMock({ [QUERYSTRING_LICENCE_KEY]: licenceKey }), generateResponseToolkitMock())
```
What do you think - do you feel it adds to the readability to break out request and params?

Reply:
Hah, yes, I initially broke those out when `request` needed to be tested as an argument but now that's no longer the case! I did consider changing it back, but personally I find it a lot more readable when the arguments are defined separately rather than inline, and for me it makes it easier to parse the tests. I'm happy to stick with whatever convention we decide though.